.dockerignore

```diff
@@ -1,7 +1,14 @@
-node_modules
 **/node_modules
-**/**/node_modules
-.git
-**/.git
-**/**/.git
+**/submodules
+**/Dockerfile
+**/.dockerignore
+**/docker-compose.yml
 .gitmodules
+**/.git
+**/docs
+**/*.md
+**/*.png
```

Comment:
Is it safe and will not exclude elements of UI in the tokenbridge UI?

Reply:
Sorry I didn't think of it, it probably will.